OtripleS.Web.Api.Tests.Unit/Services/Foundations/AssignmentAttachments/AssignmentAttachmentServiceTests.Validations.Remove.cs

```diff
@@ -18,23 +18,21 @@ public partial class AssignmentAttachmentServiceTests
         public async Task ShouldThrowValidatonExceptionOnRemoveWhenAssignmentIdIsInvalidAndLogItAsync()
         {
             // given
-            Guid randomAttachmentId = Guid.NewGuid();
+            Guid invalidAttachmentId = Guid.NewGuid();
```

Comment:
If you scope is to test invalidAttachmentId then we need that problem details as well.